vendor/github.com/ceph/go-ceph/common/admin/nfs/admin.go

```diff
@@ -1,5 +1,5 @@
-//go:build !(nautilus || octopus) && ceph_preview
-// +build !nautilus,!octopus,ceph_preview
+//go:build !(nautilus || octopus)
+// +build !nautilus,!octopus
```

Comment:
Are there still things for which we depend on `ceph_preview`? If not, remove it from the build-tags.

Reply:
@nixpanic this has to be removed from go-ceph vendor and not here ?

Reply:
I think we add the `ceph_preview` build-tag in our scripts. If we don't do that anymore, that's fine too.

Reply:
@nixpanic as mentioned here https://github.com/ceph/ceph-csi/pull/3519#issuecomment-1311491189 , the preview thing has to be checked and afacit, removing it cause the ci to fail. I will take this up as followup thing as mentioned.